Limelight/UIAppView.m

```diff
@@ -108,6 +110,19 @@ - (void) updateAppImage {
             imageView.userInteractionEnabled = YES;
             imageView.adjustsImageWhenAncestorFocused = YES;
             imageView.frame = CGRectMake(0, 0, 200, 265);
+
+            if (isCurrentGame) {
+                UIImageView *playIcon = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"Play"]];
```

Comment:
I don't really like the way our play button scales up to tvOS (and frankly it's not great on iOS either). It covers most of the app view and the contrast is not good against some box art, so it's easy to lose the play button unless you're really looking for it.

Maybe we should try doing something else to indicate the running app (maybe a green border highlight or something?).

Reply:
i added a green border highlight similar to what's now on the host selection view